user_guide_src/source/general/modules.rst

```diff
@@ -129,7 +129,7 @@ Discovery and Composer
 ======================
 
 Packages that were installed via Composer will also be discovered by default. This only requires that the namespace
-that Composer knows about is a PSR4 namespace. PSR0 namespaces will not be detected.
+that Composer knows about is a PSR-4 namespace. PSR-0 namespaces will not be detected.
```

Comment:
off-topic to this PR..

This notion on `PSR-0` is quite misleading. Composer can also detect PSR-0 namespaces via the `autoload.psr-0` key in composer.json. In fact, `mikey179/vfsstream` is using `PSR-0`. I think this should be removed in a separate PR.

Reply:
Okay, I will do.

Reply:
Wait a second. This is not about Composer, but about `$discoverInComposer` in CI4.
We need to take a look.

Reply:
It calls `$composer->getPrefixesPsr4()`, so PSR-0 namespaces will not be detected, right?
https://github.com/codeigniter4/CodeIgniter4/blob/7c08819c7e39770fb68cb3bc3aebb35a0aa32c44/system/Autoloader/Autoloader.php#L308

Reply:
Hmm. Then we need a little bit of rewording.

`Packages installed via Composer using PSR-4 namespaces will also be discovered by default. Discovery of PSR-0 namespaced packages are not currently supported.`

Reply:
PSR-0 is deprecated. We will not support it. I created a PR #5389.